docs/reference/gc-toolkit/onboarding/index.md

```diff
@@ -0,0 +1,130 @@
+
+Let’s go into https://explorer.alias.guardianconnector.net/ 
+
+#### First time login
+
+You will attempt to login. You can use a gmail account if you have one, or create an account for this service. If you do have a gmail account, it is usually easier to use that one as your login.
+
+Every time the system shows this screen, you will use the same authentication mechanism. It works across the services we provide.
+
+Your access attempt will fail, as you are not authorized yet to access the system. I will give you authorization to access now.
+
+Now you can attempt to access again.
+
+#### Overview
+
+Right now Explorer will only have one view, your current alerts.
+
+You can find more information about Explorer in our documentation: https://conservationmetrics.github.io/gc-docs/reference/gc-toolkit/gc-explorer/
+
+#### Whatsapp alerts
+
+#### Ways this space will grow
+
+As you grow in usage of the system, Explorer will host:
+
+- Your remote sensing alerts: logging, mining, airstrips and roads
+- Your views of the CoMapeo projects (both maps and media you share within it)
+
+#### Customization
+
+The way the map looks like in the alerts dashboard can be customizable.
+
+Let’s discuss what would be relevant for you to have there.
+
+### Files
+
+Files is where you can safely store your files. It is a less structured means for you to preserve your data, whatever it may be.
+Your current access credentials are:
+
+- URL: https://files.alias.guardianconnector.net/ 
+- User: 
+- Password: 
+
+You can find more information about it in our documentation: https://conservationmetrics.github.io/gc-docs/reference/gc-toolkit/filebrowser/
+
+#### Existing data
+
+You will notice that there are a few folders already available in your Files view, and that more might appear without you doing it.
+This happens because Files is also used for Guardian Connector tools to preserve data that you get from other services.
+
+Examples of this would be:
+- Alerts’ images and data
+- Media submissions done through KoboToolbox
+- Photos, videos and audios taken with CoMapeo, LocusMaps or other tools
+
+### Superset
+
+Superset is how we support you to look at your data to get insights out of it.
+
+You can get to it through this web address: https://superset.alias.guardianconnector.net/
+
+As your Data Warehouse grows, you will want to learn more about the patterns in your data. You can find more information about Superset in our documentation: https://conservationmetrics.github.io/gc-docs/reference/gc-toolkit/superset/ 
+
+Let’s give you access to it now. It will look a little empty, and that is fine for now. Once you surface the need for it, we will work with you to make good use of it.
+
+### Comapeo
+
+CoMapeo is a collaborative mapping tool that works primarily without internet connection.
+
+Guardian Connector can preserve your CoMapeo projects through its Archive Server, and make its data available for you through the tools that we’ve seen already.
```

Comment:
Here and elsewhere, link to relevant documentation in gc-docs for the tool. We discuss the archive server somewhere already.

Reply:
i.e. for each of the tools, add a link to the relevant page in gc-docs for more information.